packages/wallet/src/redux/state.ts

```diff
@@ -322,3 +335,13 @@ export function getCommitments(store: SharedData, channelId: string): Commitment
 }
 
 export { NewLedgerChannel };
+
+export function isTerminalProcessProtocolState(
+  protocolState: ProtocolState,
+): protocolState is TerminalApplicationState | TerminalFundingState | TerminalDefundingState {
+  return (
+    (isApplicationState(protocolState) && isTerminalApplicationState(protocolState)) ||
+    (isFundingState(protocolState) && isTerminalFundingState(protocolState)) ||
+    (isDefundingState(protocolState) && isTerminalDefundingState(protocolState))
```

Comment:
What about the `TerminalConcludingState` case?

Reply:
Oops, fixed!